<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	/*background-color: rgb(0, 170, 0);*/
	background-image: url(:/imagenes/fondo.png);
}

QListWidget{
	background-image: url(:/imagenes/fondo.png);
	/*background-color: rgb(0, 170, 0);*/
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="baraja" name="listWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>100</width>
      <height>360</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget{
	
	transparency=0;
	
}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="baraja" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>100</width>
      <height>360</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="baraja" name="listWidget_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>100</width>
      <height>360</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="baraja" name="listWidget_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>270</y>
      <width>100</width>
      <height>360</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget{
	
	transparency=0;
	
}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="baraja" name="listWidget_6">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>270</y>
      <width>100</width>
      <height>360</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="baraja" name="listWidget_7">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>270</y>
      <width>100</width>
      <height>360</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="baraja" name="listBaraja">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>100</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="baraja" name="listBaraja_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>100</width>
      <height>130</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="baraja" name="listBaraja_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>100</width>
      <height>130</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="baraja" name="listBaraja_4">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>40</y>
      <width>100</width>
      <height>130</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="baraja" name="listWidget_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>270</y>
      <width>100</width>
      <height>360</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget{
	
	transparency=0;
	
}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="baraja" name="listBaraja_5">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>40</y>
      <width>100</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>100</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/imagenes/reves.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1002</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuJuego">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="actionABOUT"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuJuego"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNuevo">
   <property name="text">
    <string>Nuevo</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionABOUT">
   <property name="text">
    <string>ABOUT...</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>baraja</class>
   <extends>QListWidget</extends>
   <header location="global">baraja.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
